.dockerignore

```diff
@@ -1,8 +0,0 @@
-node_modules
```

Comment:
Unintentional?

Reply:
do we really need a `.dockerignore` if we're not packaging this as a container image?

Reply:
Yes because that means we ignore Docker /s

Reply:
yeah it is fine if we want to delete